tests/test_queries.py

```diff
@@ -1148,217 +1209,282 @@ def test_numbers():
     )
     assert number_to_neutral(241000000000) == "tvö hundruð fjörutíu og einn milljarður"
     assert number_to_neutral(100000000) == "eitt hundrað milljónir"
+    assert number_to_neutral(1001000000) == "einn milljarður og ein milljón"
+    assert number_to_neutral(1002000000) == "einn milljarður og tvær milljónir"
     assert number_to_neutral(200000000000) == "tvö hundruð milljarðar"
     assert (
         number_to_neutral(10000000000000000000000000000000000000000000000000000000)
         == "tíu milljónir oktilljóna"
     )
+    assert (
+        number_to_neutral(1000000000000000000000000000000000000001000000000)
+        == "ein oktilljón og einn milljarður"
+    )
+    assert (
+        number_to_neutral(1000000000000000000000000000000000000003000000000)
+        == "ein oktilljón og þrír milljarðar"
+    )
+    assert number_to_neutral(3000400000) == "þrír milljarðar og fjögur hundruð þúsund"
+    assert (
+        number_to_neutral(2000000000000000000000000000000000100000000000000)
+        == "tvær oktilljónir og eitt hundrað billjónir"
```

Comment:
Love that you're also handling these enormous and infrequently used numbers.

Reply:
These tests were for the correct placement of "og", but good to be ready for everything :sunglasses: